sql/src/main/java/io/crate/execution/dsl/projection/builder/InputColumns.java

```diff
@@ -138,11 +151,25 @@ public Symbol visitFunction(Function symbol, final SourceSymbols sourceSymbols)
         if (replacement != null) {
             return replacement;
         }
-        ArrayList<Symbol> args = new ArrayList<>(symbol.arguments().size());
-        for (Symbol arg : symbol.arguments()) {
+        return null;
+    }
+
+    private ArrayList<Symbol> replaceFunctionArgs(Function function, SourceSymbols sourceSymbols) {
```

Comment:
maybe name this `getProccessedArgs` or something? I initially assumed that this might mutate something due to the `replace`. Could probably also pass in the arguments directly to clarify that nothing else of the function is used.